build-system/app.js

```diff
@@ -1118,13 +1118,13 @@ function replaceUrls(mode, file, hostName, inabox, storyV1) {
         hostName + '/dist/amp-shadow.js');
     file = file.replace(
         /https:\/\/cdn\.ampproject\.org\/amp4ads-v0\.js/g,
-        hostName + '/dist/amp-inabox.js');
+        hostName + '/dist/amp-inabox-lite.js');
```

Comment:
done

Reply:
Thanks. I would say please document it. But then I realize it's not necessary since we don't expect anyone to test with the lite version other than development purpose And later on we'll simply remove the version and always replace with lite version : ) LGTM.